packages/components/icon/src/icon.tsx

```diff
@@ -27,7 +28,7 @@ const fallbackIcon = {
 
 export interface IconProps
   extends Omit<React.SVGAttributes<SVGElement>, keyof ChakraProps>,
-    ChakraProps {}
+    Omit<ChakraProps, "orientation"> {}
```

Comment:
Why the need for this? Just curious

Reply:
@segunadebayo it clashes with the typings for `styleConfig` decided to omit it since Icon doesn't need it.